src/pages/Home.js

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import GoogleLogin from 'react-google-login';
+
+const Home = () => {
+const [loginData, setLoginData] = useState(
+    localStorage.getItem('loginData')
+      ? JSON.parse(localStorage.getItem('loginData'))
+      : null
+  );
+
+  const handleFailure = (result) => {
+    alert(result);
+  };
+  const handleLogin = async (googleData) => {
+    console.log(googleData);
+    const res = await fetch('/api/google-login', {
+      method: 'POST', 
+      body: JSON.stringify({
+        token: googleData.tokenId,
+      }),
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    });
+
+    const data = await res.json();
+    console.log('res', data);
+    setLoginData(data);
+    localStorage.setItem('loginData', JSON.stringify(data));
+  };
+  const handleLogout = () => {
```

Comment:
you can extract this method in another file as it can be a function that can be reused in other components.